packages/react-native-codegen/src/parsers/flow/parser.js

```diff
@@ -89,7 +96,21 @@ class FlowParser implements Parser {
   parseFile(filename: string): SchemaType {
     const contents = fs.readFileSync(filename, 'utf8');
 
-    return buildSchema(contents, filename, this);
+    return buildSchema(
+      contents,
+      filename,
+      wrapComponentSchema,
+      buildComponentSchema,
+      buildModuleSchema,
+      Visitor,
+      this,
+    );
+  }
```

Comment:
Perhaps it is for the next PR, but should we move the `parseString` and `parseModuleFixture` in the `parser` and we can get rid of the `flow/index.js` and `typescript/index.js` files... 🤔

With this change, all the methods may resolve to call `parseString` instead of having their own call to `buildSchema` and we can isolate that call. What do you think?

Reply:
Yes, I tried to use `parseString` in `parseFile` but there is a circular dependency: parser.parseFile -> parseString -> buildSchema with parser in parameters.

To be able to do this, we need to move `parseModuleFixture` and `parseString` in the parser as you suggested. Maybe we can merge `parseString` and `parseFile` when it's done since they are practically identical. But, I think that is outside the scope of this PR and we should create a new PR when this one is merged.

Is it good for you?

Reply:
Yes, we can do it in a future pr